scripts/local_test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
I know we've talked about that it'd be great to be able to run integration test from local dev machine. And I assume this is moving towards that direction? 

We have two goals here: 1) make integration test more realistic 2) try to make the integration runnable from local dev machine. Let's try to achieve them separately because 1 is more important than 2.

Could we make sure things are working fine in GH workflow _first_ before we try to make the integration test runnable locally? 

I'm also questioning the decision to put things into script, especially the build part. Let's have a little more discussion before we take action.

Reply:
Synced offline, breaking down the PR...